lib/internal/test_runner/coverage.js

```diff
@@ -24,16 +24,19 @@ const {
   opendirSync,
   readFileSync,
 } = require('fs');
-const { setupCoverageHooks } = require('internal/util');
+const { setupCoverageHooks, getLazy } = require('internal/util');
+const { getOptionValue } = require('internal/options');
 const { tmpdir } = require('os');
-const { join, resolve } = require('path');
+const { join, resolve, relative, matchesGlob } = require('path');
 const { fileURLToPath } = require('internal/url');
 const { kMappings, SourceMap } = require('internal/source_map/source_map');
 const kCoverageFileRegex = /^coverage-(\d+)-(\d{13})-(\d+)\.json$/;
 const kIgnoreRegex = /\/\* node:coverage ignore next (?<count>\d+ )?\*\//;
 const kLineEndingRegex = /\r?\n$/u;
 const kLineSplitRegex = /(?<=\r?\n)/u;
 const kStatusRegex = /\/\* node:coverage (?<status>enable|disable) \*\//;
+const lazyExcludeGlobs = getLazy(() => getOptionValue('--coverage-exclude-glob') ?? []);
+const lazyIncludeGlobs = getLazy(() => getOptionValue('--coverage-include-glob') ?? []);
```

Comment:
why lazy load this? I think this value is already in memory, lazy loading should not have any benifit here